babel/numbers.py

```diff
@@ -416,15 +429,54 @@ def format_decimal(
                                  the format pattern. Defaults to `True`.
     :param group_separator: Boolean to switch group separator on/off in a locale's
                             number format.
+    :param format_type: Format to use (`None`, "short" or "long"). The standard decimal
+                        format is `None`. Defaults to `None`.
+    :param compact_fraction_digits: Number of fraction digits to use in "short" or "long"
+                                    format. Defaults to `0`. If this is set to
+                                    a value greater than `0`, the `decimal_quantization`
+                                    will be treated as `False`.
```

Comment:
I'm a bit on the fence about these being additional arguments to `format_decimal`, especially given `format_type` being such a generic term.

A separate function – `format_compact_decimal(number, *, type, digits=0, locale=None)` might be a better API? (Would there be a reason to keep `decimal_quantization` and `group_separator` in that API?)

Reply:
I've seen other APIs that only have one method for number formatting, for example [GlobalizeJS](https://github.com/globalizejs/globalize#number-module). I agree that making it separate would probably be cleaner since some of the parameters are unused.

Reply:
I've updated the API, let me know what you think :+1: